chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/ics/IcsErrorsReportEntity.java

```diff
@@ -14,22 +14,20 @@
 import gov.healthit.chpl.entity.CertificationBodyEntity;
 import lombok.Data;
 
-/**
- * Entity containing needed data for Listings with ICS errors.
- * @author alarned
```

Comment:
Yay! I'm surprised I still have any of these lines in the codebase. Happy to see them go away :)